### ISSUE TITLE
Default repositories should use HTTPS URLs

### ISSUE THREAD

**Issue by nodiscc** [tim-lebedkov/npackd-cpp#8]
Since https://npackd.appspot.com/ properly supports HTTPS, default repositories in the application should use `https://` URLs instead of `http`. Links on https://npackd.appspot.com/rep should also be updated to point to `https://` URLs. This gives better guarantees regarding the integrity/authenticity of repositories.

**Comment by tim-lebedkov** [comment]
The default repositories stored in the binary use "https": https://github.com/tim-lebedkov/npackd-cpp/blob/master/wpmcpp/src/abstractrepository.cpp#L869

https://npackd.appspot.com/rep also points to "https" repositories. 

The only exception is http://npackd.appspot.com/rep

**Comment by nodiscc** [comment]
Sorry, I got confused after removing  the default repositories as described in https://github.com/tim-lebedkov/npackd-cpp/issues/7; I ended up on http://npackd.appspot.com/rep and noticed the plain-http URLs, assumed it was the default.

So the only problem is with http://npackd.appspot.com/rep. Seeing that all search engines point to the HTTPS version it's probably not a big problem. I manually entered `npackd.appspot.com` in my browser address bar and ended up getting the HTTP version.

Thanks
